aws/rust-runtime/aws-config/test-data/profile-provider/credential_process/fs/home/.aws/config

```diff
@@ -0,0 +1,6 @@
+[default]
```

Comment:
can we add a test where the command is something like `exit 1`? And a test where it returns invalid JSON?